src/components/templates/Jazzbar/components/JazzBarRoom/Room.scss

```diff
@@ -1,10 +1,90 @@
 @import "scss/constants.scss";
+@import "../../jazzbar-constants.scss";
 
-.jazzbar-participant-container {
-  background: $black;
-  display: flex;
-  justify-content: center;
-  width: 100%;
+.jazzbar-room {
+  &__participant {
+    background-color: $light-grey;
+    display: flex;
+    justify-content: center;
+    justify-self: center;
+    max-width: $video-participant-optimal-width;
+    max-height: $video-participant-optimal-height;
+    border-radius: 15px;
+
+    .participant {
+      display: flex;
+      height: 100%;
+      padding: 0;
+      border-radius: 22px;
+
+      video {
+        height: 100%;
+        width: 100%;
+        border-radius: 15px;
+      }
+
+      .mirrored {
+        transform: scale(-1, 1);
+      }
+
+      .profile-icon {
+        position: absolute;
+        bottom: 0;
+        left: 0;
+        width: 25px;
+        height: 25px;
+        border-radius: 188%;
+        z-index: z(jazzbar-participant-profile-icon);
+      }
+
+      .mute-container {
+        position: absolute;
+        bottom: 10px;
+        left: 0;
+        right: 0;
+        text-align: center;
+        display: flex;
+        justify-content: center;
+      }
+
+      .mute-other-container {
+        position: absolute;
+        top: 10px;
+        right: 10px;
+        text-align: center;
+        display: flex;
+        justify-content: center;
+      }
+    }
+  }
+
+  &__participants {
+    display: flex;
+    grid-area: items;
+
+    .jazzbar__participant {
+      margin: 0 0.25rem;
+      width: $video-participant-optimal-width;
+    }
+  }
+
+  @media (min-width: $small-laptop-width) {
+    &__participants {
+      justify-content: center;
+      .jazzbar__participant {
+        max-width: 24%;
+      }
+    }
+  }
+
+  @media (max-width: $small-laptop-width) {
+    &__participants {
+      justify-content: space-around;
+      .jazzbar__participant {
+        max-width: 49%;
+      }
+    }
+  }
```

Comment:
I don't think `&__participants` will refer to the same thing here, I think it will only 'reach' 1 level up, so you may need to use the full name for these styles.

Reply:
Also, should `.jazzbar__participant` be `.jazzbar-room__participant`?

Reply:
> I don't think &__participants will refer to the same thing here, I think it will only 'reach' 1 level up, so you may need to use the full name for these styles.

I have just checked. It applies the rules correctly 

![image](https://user-images.githubusercontent.com/63194656/109815099-f1ff0680-7c37-11eb-9805-52b1493443a3.png)

Reply:
> Also, should .jazzbar__participant be .jazzbar-room__participant?

changed in https://github.com/sparkletown/sparkle/pull/1217/commits/002f1b4686eae7db7a910850d16dde1407bc4d2c